darts/dataprocessing/transformers/fittable_data_transformer.py

```diff
@@ -50,14 +69,20 @@ def __init__(
         amount of data. Using instance methods would imply copying the instance's data through multiple processes, which
         can easily introduce a bottleneck and nullify parallelisation benefits.
         """
-        super().__init__(name=name, n_jobs=n_jobs, verbose=verbose)
+        super().__init__(
```

Comment:
shouldn't the `super().__init__()` be called after instantiating the other attributes or is it valid only for the transforms that directly inherit from `BaseDataTransformer`?

Reply:
My understanding is that this is necessary in the childmost classes only (where the instance variables are defined).